test/info-test.js

```diff
@@ -104,6 +104,19 @@ describe('ytdl.getInfo()', () => {
       });
     });
 
+    describe('With IPv6 Block', () => {
+      it('Sends request with IPv6 address', async() => {
+        const scope = nock(id, {
+          type: 'vevo',
+          player: true,
+        });
+        let info = await ytdl.getInfo(id, { IPv6Block: '2001:2::/48' });
+        await nock.url(info.formats[0].url).reply(function checkAddr() {
+          if (this.req.options.localAddress.includes(':')) scope.done();
+        });
+      });
+    });
+
```

Comment:
"without ipv6" would be cool
i know that the default socket defaults to pad addresses to ipv6
e.g.:
`localhost` => `::1`
`127.0.0.1` => `::ffff:127.0.0.1`
`10.0.10.59` => `::ffff:10.0.10.59`

[net#isIPv6](https://nodejs.org/api/net.html#net_net_isipv6_input) should do the trick

Reply:
so like replace the `.includes` part to `net#isIPv6` right

Reply:
👍

Reply:
done, did a typo in commit `if -> of` but i guess its fine now